### DISCUSSION TITLE
Windows can't find chan-downloader

### DISCUSSION THREAD

**Discussion by UltimateGattai** [mariot/chan-downloader#12] [Q&A]
I'm new to using Github and I'm trying to use Mariot/Chan-downloader, I downloaded the code and placed the bat file into the same folder as the code but "Windows can't  Chan-Downloader" error comes up. What have I done wrong?
![Chan Downloader error](https://user-images.githubusercontent.com/31506227/145567636-5c186be3-1be7-4bd3-b34d-8cb454327450.png)

**Comment by mariot** [comment]
Hi! Can you try to directly use the release files instead of the code? Here are the releases: https://github.com/mariot/chan-downloader/releases
It contains the exe file.

**Reply by mushoo22** [reply]
Hi Mariot, 
So I've downloaded the .exe file and place it in the same folder as my bat file.
When I paste the URL into the command prompt, it just closes without doing anything.
Any advise?
Also, will this work for archived.moe?
